### PR TITLE
Apple clone page

### PR DESCRIPTION
**Pull request by cyonii**
Complete clone of Apple's 2014 homepage

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        <a class="nav-item" href="#">iPhone</a>
+        <a class="nav-item" href="#">iPad</a>
+        <a class="nav-item" href="#">iTunes</a>
+        <a class="nav-item" href="#">Support</a>
+        <form class="nav-item" action="index.html" method="post">
+          <input class="form-control" type="text" name="search">
+        </form>
+      </nav>
+
+      <!-- <h1 class="jumbotron-header">What will your verse be?</h1> -->
```
Comment (kevcap) [inline index.html]: Remove this commented piece of code.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      <img width="1000" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/your_verse_title_small_2x.png" alt="Jumbotron title">
+      <img width="900" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/your_verse_subhead_small_2x.png" alt="Jumbotron subtitle">
```
Comment (kevcap) [inline index.html]: Remove all inline styling.

**Review by kevcap — CHANGES_REQUESTED** [review]
# [STATUS] CHANGES REQUIRED♻️

  
#### Hey team, Is a pleasure to review such a good project! Congrats for the amazing work you guys already did, the project looks really nice and I can see that you guys are working hard on it, and we're almost done, there are just *three* minor changes that you guys have to implement to proceed:

## [REQUIRED CHANGES]

- [x] Please delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
:x: **commented piece of codes** /* `.nav { color: red;}` */  
:heavy_check_mark: **comments about code** /*Navigation */

- [x] You have inline styling, please remove all it. 🤓

- [x] Add a meaningful description to your pull request. For more info check [here](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/)
Tip: Describe what you guys did in this project; Like: We used CSS flexbox, float and grid; we added a sidebar, we used semantic HTML. In real world: PR Should be descriptive, so the reviewer can know what you guys are trying to change at the main project.

## Summary

✨ Your CSS looks really organized and professional! 👨‍💻
✨ The project looks really nice 💯
✨ You guys got my attention, I'll keep my eyes on you! Keep up the good work! 🐱‍👓
✨ Good job with that project, it looks really good for a first review 💥
✨ Again, wow, what a project! 🚀



## Please make the required changes and resubmit for a new code review once you are ready💪

## As always, I wish you all the very best and see you both at the top🚀

  👨🏻‍💻 **TSE: Kevin Alves**

    

  📫 Have any question? Better call Kevin:

  <a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

    
  <sub>*We're here to help you, to ensure that you're using the best practices and getting you ready for the job market. In real-world, you'll have a lot of changes requests for your real clients, it's good to get used to it.</sub>

**Comment by cyonii** [comment]
Thank you so much for such a wonderful and positive comment. We will make all required changes ASAP

**Comment by kevcap** [comment]
### Great! Keep up the good work guys!  🚀

**Review by ghost — APPROVED** [review]
# Status: Project Approved

## Hello team,

**Congratulations and good luck on your next project 😃.**

**PS: You can reach me on slack(Rose Wanjohi).**

**Comment by cyonii** [comment]
Thank you

**Pull request merged**